### PR TITLE
Cleanup

### PR DESCRIPTION
**Pull request by TheCoolBoss**
- Removed unused files
- Reformats output to a more readable form
- Continuously asks user for another city if destination is the same as source

### CODE REVIEW

**Review by dnguyen09 — APPROVED** [review]
LGTM!

File: src/main/java/com/solvd/block3/ui/UiUtils.java [hunk 1]
```diff
@@ -92,34 +91,34 @@ public static boolean validateCity(String cityId) {
...
-            LOGGER.info("Invalid input.");
-            LOGGER.info("Enter 's' to get the shortest path or 'c' for the cheapest");  
-            mode = SCANNER.nextLine().trim();
-        }
 
-        return mode;
+        // Use regex to match variations of input
+        if (Pattern.matches("(?i)^s(hort(est)?)?$", mode)) {
+            return "s";
+        } else if (Pattern.matches("(?i)^c(heap(est)?)?$", mode)) {
```
Comment (dshaur) [inline src/main/java/com/solvd/block3/ui/UiUtils.java:101]: Added pattern to match potential variations of user input for cheapest and shortest routes.

File: src/main/java/com/solvd/block3/ui/UiUtils.java [hunk 2]
```diff
@@ -92,34 +91,34 @@ public static boolean validateCity(String cityId) {
...
         String mode = SCANNER.nextLine().trim();
-        while (!mode.equalsIgnoreCase("s") && !mode.equalsIgnoreCase("c"))
-        {
-            LOGGER.info("Invalid input.");
-            LOGGER.info("Enter 's' to get the shortest path or 'c' for the cheapest");  
-            mode = SCANNER.nextLine().trim();
-        }
 
-        return mode;
+        // Use regex to match variations of input
```
Comment (dshaur) [inline src/main/java/com/solvd/block3/ui/UiUtils.java:98]: Added a pattern to match potential variations of user input for the cheapest and shortest flight options.

**Review by dshaur — APPROVED** [review]
Looks good to me. I added some minor fixes to the output wording and used regex patterns.

**Review by JuicingJuicer — APPROVED** [review]
Looks good

**Pull request merged**